config/initializers/senic.rb

```diff
@@ -0,0 +1,3 @@
+Scenic.configure do |config|
+  config.database = Scenic::Adapters::Mysql.new
```

Comment:
Can we avoid saying MySQL here and pull it from `Rails.config` or `database.yml`? I'm thinking of barnacles, the other sister sites using postgresql, and my daydream of moving Lobsters to pg. I saw that `Gemfile` gets `scenic-mysql` added; anyone using pg now is already forked there, just hoping to minimize their edit points.